vips/vip-304/bscmainnet.ts

```diff
@@ -0,0 +1,81 @@
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const VUSDT = "0xfD5840Cd36d94D7229439859C0112a4185BC0255";
+export const VUSDC = "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8";
+export const VDAI = "0x334b3eCB4DCa3593BCCC3c7EBD1A1C1d1780FBF1";
+export const VFDUSD = "0xC4eF4229FEc74Ccfe17B2bdeF7715fAC740BA0ba";
+export const VTUSD = "0xBf762cd5991cA1DCdDaC9ae5C638F5B5Dc3Bee6E";
+export const VUSDT_STABLE_COIN = "0x5e3072305F9caE1c7A82F6Fe9E38811c74922c3B";
+export const VUSDT_GAMEFI = "0x4978591f17670A846137d9d613e333C38dc68A37";
+export const VUSDT_DEFI = "0x1D8bBDE12B6b34140604E18e9f9c6e14deC16854";
+
+export const NEW_VUSDT_IR = "0xBe4609d972FdEBAa9DC870F4A957F40C301bEb1D";
+export const NEW_VUSDC_IR = "0xBe4609d972FdEBAa9DC870F4A957F40C301bEb1D";
+export const NEW_VDAI_IR = "0xBe4609d972FdEBAa9DC870F4A957F40C301bEb1D";
```

Comment:
I thought I remembered a conversation about 1 IR contract per market. Should these markets have their own IR even though they share parameters?

Reply:
nit: if we name IR models with their params it makes it could make it easier to review